patches/minecraft/net/minecraft/world/entity/ai/goal/RangedCrossbowAttackGoal.java.patch

```diff
@@ -1,5 +1,33 @@
 --- a/net/minecraft/world/entity/ai/goal/RangedCrossbowAttackGoal.java
 +++ b/net/minecraft/world/entity/ai/goal/RangedCrossbowAttackGoal.java
+@@ -12,7 +_,7 @@
+ import net.minecraft.world.item.ItemStack;
+ import net.minecraft.world.item.Items;
+ 
+-public class RangedCrossbowAttackGoal<T extends Monster & RangedAttackMob & CrossbowAttackMob> extends Goal {
++public class RangedCrossbowAttackGoal<T extends net.minecraft.world.entity.Mob & CrossbowAttackMob> extends Goal {
+    public static final UniformInt f_25804_ = TimeUtil.m_145020_(1, 2);
+    private final T f_25805_;
+    private RangedCrossbowAttackGoal.CrossbowState f_25806_ = RangedCrossbowAttackGoal.CrossbowState.UNCHARGED;
+@@ -22,10 +_,14 @@
+    private int f_25810_;
+    private int f_25811_;
+ 
+-   public RangedCrossbowAttackGoal(T p_25814_, double p_25815_, float p_25816_) {
+-      this.f_25805_ = p_25814_;
+-      this.f_25807_ = p_25815_;
+-      this.f_25808_ = p_25816_ * p_25816_;
++   public <M extends Monster & CrossbowAttackMob> RangedCrossbowAttackGoal(M p_25814_, double p_25815_, float p_25816_) {
++      this((T) p_25814_, p_25815_, p_25816_);
++   }
++
++   public RangedCrossbowAttackGoal(T pMob, double pSpeedModifier, float pAttackRadius) {
```

Comment:
This needs an exc entry added to https://github.com/neoforged/NeoForge/blob/1.20.x/src/main/resources/forge.exc

Reply:
To add to that, please also remove the `p` prefixes of the parameter names.

Reply:
269254f10753f1bf521121cc57a84db258535c6b fixes this.